src/packet/payload.rs

```diff
@@ -43,7 +44,8 @@ impl Payloader {
             let last = idx == len - 1;
 
             let previous_data = stream.last_packet();
-            let marker = self.pack.is_marker(data.as_slice(), previous_data, last);
+            let marker = self.pack.is_marker(data.as_slice(), previous_data, last)
+                || (is_audio && start_of_talk_spurt);
```

Comment:
Is using the marker for `start_of_talk_spurt` something that is the same across all audio codecs?

Reply:
Yes I believe we should use start of talkspurt for all audio codecs.

One could argue that you can add logic to figure out when to add a marker bit for other codecs, if str0m notices a silent period. (timestamp gap, but no seq number gap or comfort noise payload types)

But imho I think it's fine leaving it to the caller for all codecs.

References that I found:
https://datatracker.ietf.org/doc/html/rfc3551#section-4.1
https://datatracker.ietf.org/doc/html/rfc3389#section-5.1

Reply:
cool